main.go

```diff
@@ -1,4 +1,30 @@
 package main
 
+import (
+	"fmt"
+	"github.com/log-yarder/yarder/appender"
+	"github.com/log-yarder/yarder/storage"
+	"io/ioutil"
+	"path"
+)
+
+const (
+	maxEntriesPerChunk = 10
+)
+
 func main() {
+	tmpDir, err := ioutil.TempDir(path.Join("/", "tmp"), "yarder-dev")
+	if err != nil {
+		panic(fmt.Sprintf("Unable to create temp dir, %v", err))
```

Comment:
Use the `log` package rather than panicking.

Reply:
Done.